app/views/form_items/_printable.html.erb

```diff
@@ -1,10 +1,10 @@
-<% items.each do |item, subitems| %>
-  <% if item.is_a?(QingGroup) %>
-    <% if !subitems.empty? %>
-      <%= render "qing_groups/printable", qing_group: item %>
```

Comment:
Why did this move out of the `if` block? I think if there are no children, we don't want to render the group at all.